src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc

```diff
@@ -91,13 +91,13 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
 :===
 
 
-.Example `meta.yaml` for a Che-Theia plug-in: the Che machine-exec Service
+.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
 [source,yaml]
 ----
 apiVersion: v2
 category: Other
-description: Che Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
-displayName: Che machine-exec Service
+description: Che Plugin with che-machine-exec service to provide creation terminal or tasks for {prod} workspace containers.
```

Comment:
This sentence is giving me a headache :(

Reply:
It is a verbatim copy of a real metadata, I am sorry :/. Should we review them?